app/src/main/java/com/cornellappdev/android/eatery/ui/screens/FavoritesScreen.kt

```diff
@@ -170,3 +271,122 @@ fun EateryBlob(
         color = GrayTwo
     ) {}
 }
+
+
+@Composable
+fun FilterHeader(
+    filters: List<Filter>,
+    onFilterClicked: (Filter) -> Unit
+) {
+    ItemFavoriteFilterRow(
+        currentFiltersSelected = filters,
+        onFilterClicked = onFilterClicked,
+    )
+}
+
+
+data class ItemFavoritesCardViewState(
+    val itemName: String,
+    val availability: EateryStatus,
+    val mealAvailability: Map<String, List<String>>
+)
+
+@Composable
+fun ItemFavoritesCard(viewState: ItemFavoritesCardViewState) {
+    var isExpanded by remember { mutableStateOf(false) }
+    val rotation: Float by animateFloatAsState(
+        if (isExpanded) 180F else 0F,
+        label = "chevron rotation"
+    )
+    Card(
+        modifier = Modifier
+            .fillMaxWidth()
+            .border(BorderStroke(Dp.Hairline, GrayZero), RoundedCornerShape(8)),
+        shape = RoundedCornerShape(8.dp),
+        colors = CardDefaults.cardColors(
+            containerColor = Color.White
+        ),
+        elevation = CardDefaults.cardElevation(8.dp)
+    ) {
+        Column(
```

Comment:
This file's getting a bit long LOL, can you pull this out into a separate file under the `components` directory somewhere?